integration-tests/emulated/tests/assets/asset-hub-kusama/src/tests/mod.rs

```diff
@@ -26,3 +26,15 @@ emulated_integration_tests_common::include_penpal_create_foreign_asset_on_asset_
 	KUSAMA_ED,
 	system_parachains_constants::kusama::fee::WeightToFee
 );
+
+#[test]
+fn assets_can_be_claimed() {
```

Comment:
@acatangiu @franciscoaguirre 
And another thing, as Cisco pointed out in DM, we didn't add the integration tests to the relays. So, perhaps this could be a good point to start, with these tests as the first ones. (Also, add the same to polkadot-sdk.)
Wdyt?

Reply:
also we have tests like `relay_sets_system_para_xcm_supported_version` which uses just Relay chain and are copied to every parachain tests, so we could just have them one in the relay tests

Reply:
I think it's a good idea to start making tests for the relays for functionality that is entirely theirs, like this `claim_assets` extrinsic. I'll create them with just this test for now if that sounds good

Reply:
hmm, maybe for these claim_assets test we can move them directly to the relay runtime tests: https://github.com/bkontur/runtimes/pull/7#discussion_r1511084890

we don't need integration tests for tests that requires only one runtime

Reply:
Hmm I thought of it as an "integration test" because it's already integrating a lot of things in the runtime. But it's true integration tests here refer to integrating with more than one runtime. I could move them to be unit tests

Reply:
I say we consolidate all these compiled tests running against actual runtimes under `integration-tests/emulated` even if some would also work as runtime unit-tests. See https://github.com/bkontur/runtimes/pull/7#discussion_r1511248780

Reply:
So would you create a test under `integration-tests/emulated/tests/relay/` for this?